### PR TITLE
Add geo tag

### LINKED ISSUE #42: Add Locational Information to tweet in storage
Add geo location where available to the Tweets that are stored when a search query is made

### PR DESCRIPTION
**Pull request by VasilyShelkov**
-Added Place, Country nodes and edges to Scheme, Objects and Insert Actions
-Now Processes Place and added latitude/longitude to tweet
-Added exampleTwitterJson for quick example json retreival

closes #42

### CODE REVIEW

**Comment by VasilyShelkov** [comment]
image

is the structure of the locational info

File: src/server/twitterSearch.js [hunk 1]
```diff
@@ -25,16 +26,39 @@ export const TwitAccess = new Twit({
...
+  const latLong = findLatitudeLongitude(rawTweet);
   return Builders.TweetBuilder()
     .id(rawTweet.id_str)
     .content(rawTweet.text)
     .date(moment(new Date(rawTweet.created_at)).format('YYYY-MM-DD HH:mm:ss'))
-    .likes(rawTweet.favourites_count || 0)
-    .retweets(rawTweet.retweet_count || 0)
+    .likes(rawTweet.favorite_count)
+    .retweets(rawTweet.retweet_count)
+    .latitude(latLong[0])
```
Comment (Winwardo) [inline src/server/twitterSearch.js]: I'd prefer us to return an object with specific keys than an array really. Your call here

**Comment by Winwardo** [comment]
@VasilyShelkov please update from master. I've left a single comment.

Looks great! :+1:

**Comment by VasilyShelkov** [comment]
@Winwardo, changed latLong[i] into object ready for merge, fixes #42

**Comment by VasilyShelkov** [comment]
eh was it not merged with Master branch ? Gitkraken was saying it was up to date with master :S

**Pull request merged**